.github/release/full/Dockerfile

```diff
@@ -3,11 +3,16 @@ FROM debian:bookworm-slim
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
-# Install HDF5 bindings for Python, and a few other dependencies.
-RUN apt-get update && \
-    apt-get --yes --no-install-recommends --no-install-suggests install \
-      ca-certificates python-is-python3 python3-h5py python3-pip python3-wheel python3-venv && \
-    rm -rf /var/lib/apt/lists/* && rm -rf /var/cache/apt
+# Install build prerequisites.
+RUN rm -f /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+RUN \
+    --mount=type=cache,id=apt,sharing=locked,target=/var/cache/apt \
+    --mount=type=cache,id=apt,sharing=locked,target=/var/lib/apt \
+    true \
```

Comment:
What is true doing in this context?

Reply:
It's essentially a noop, and is just there to make the subsequent lines all start with `&&`. Otherwise, there would be a hangover.